thunder/tests/test_recipes.py

```diff
@@ -7,9 +7,12 @@
 
 from thunder.extend import deregister_executor
 from torch.testing import assert_close, make_tensor
+from thunder.recipes import HFTransformers
+from thunder.executors import nvfuser_available
 from thunder.tests.framework import version_between, IS_WINDOWS
 
 
+@pytest.mark.skipif(not nvfuser_available(), reason="NVFuser is not available")
```

Comment:
```suggestion
@pytest.mark.skipif(not nvfuser_available(), reason="nvFuser is not available")
```
similarly for the other cases